src/lib/components/self-hosted/view-all-features.svelte

```diff
@@ -7,7 +7,7 @@
 
 <SectionCommon
   title="View all features"
-  text="Install Gitpod Self-Hosted on <a href='/docs/self-hosted/latest/installation-guides'>GKE</a>, <a href='/docs/self-hosted/latest/installation-guides'>EKS</a>, <a href='/docs/self-hosted/latest/cluster-set-up/on-microsoft-aks'>AKS</a>, or <a href='/docs/self-hosted/latest/cluster-set-up/on-k3s'>K3s</a>."
+  text="Install Gitpod Self-Hosted on <a href='/docs/self-hosted/latest/installation-guides'>GKE</a>, <a href='/docs/self-hosted/latest/installation-guides'>EKS</a> or <a href='/docs/self-hosted/latest/cluster-set-up/on-microsoft-aks'>AKS</a>."
```

Comment:
```suggestion
  text="Install Gitpod Self-Hosted on <a href='/docs/self-hosted/latest/installation-guides'>GKE</a>, <a href='/docs/self-hosted/latest/installation-guides'>EKS</a>, or <a href='/docs/self-hosted/latest/cluster-set-up/on-microsoft-aks'>AKS</a>."
```

tiny suggestion for consistency